src/ReactWrapper.jsx

```diff
@@ -722,11 +722,19 @@ class ReactWrapper {
    * @param {*} initialValue - the initial value
    * @returns {*}
    */
-  reduce(fn, initialValue) {
-    return this.getNodes().reduce(
-      (accum, n, i) => fn.call(this, accum, this.wrap(n), i),
-      initialValue,
-    );
+  reduce(fn) {
```

Comment:
could we maybe add the old argument back within comment blocks so we see that it is part of the API. Yes jsdoc helps above, but it gets a bit confusing when see this and then L729